_config.yml

```diff
@@ -23,5 +23,7 @@ navigation:
   url: documentation.html
 - text: Community
   url: contribute.html
+- text: Blog
+  url: blog.html
```

Comment:
I tried putting an external url there, but our layout causes it to be rendered as "http://theforeman.org/http://blog.theforeman.org". I couldn't see an easy way of fixing the loop over site.navigation in the navigation layout, so a separate handler seemed easier.